setup.py

```diff
@@ -25,7 +25,7 @@
     entry_points={
         'console_scripts': [
             'vivbin=vivisect.vivbin:main',
-            'vdbbin=vdbbin.vdbbin:main',
+            'vdbbin=vdb.vdbbin:main',
```

Comment:
this is downright embarrassing.  this has been broken since we changed and supported PyPi